librz/core/cmd_print.c

```diff
@@ -1592,6 +1582,80 @@ static void cmd_print_gadget(RzCore *core, const char *_input) {
 	}
 }
 
+RZ_IPI RzCmdStatus rz_cmd_print_timestamp_unix_handler(RzCore *core, int argc, const char **argv) {
+	// len must be multiple of 4 since rz_mem_copyendian move data in fours - sizeof(ut32)
+	st64 l;
+	ut8 *block = core->block;
+	int len = core->blocksize;
+	if (len < sizeof(ut32)) {
+		eprintf("You should change the block size. Please run `b %d\n`", (int)sizeof(ut32));
```

Comment:
I suggest to use `%zu` here instead of casting to int. Also I recommend to reword the message to hint that any value *bigger* than `sizeof(ut32)` would suffice, ut also that it should be a multiply of it.

Reply:
```suggestion
		eprintf("Please change the block size to a value greater than and a multiple of 4. For example,
		 	run `b %zu\n`", sizeof(ut32));

```

Reply:
@officialcjunior sounds good.

Reply:
Though instead of hardcoding `4` it makes sense to reuse `%zu` here too.